test/JDBC/expected/binary-index-vu-verify.out

```diff
@@ -0,0 +1,132 @@
+GO
+
+
+
+exec babel_3939_vu_prepare_p1;
+GO
+~~START~~
+text
+Query Text: EXEC babel_3939_vu_prepare_p1
+  Query Text: select * from tab_binary where a = 0xBAADF00D
+  ->  Index Scan using ix_tab_binary_atab_binary30474ea5eaee4ec0e0a5a86377abb1ac on tab_binary
+        Index Cond: ((a)::bbf_binary = '0xbaadf00d'::bbf_varbinary)
+  Query Text: select * from tab_binary where a = cast (0xBAADF00D as  binary )
+  ->  Index Scan using ix_tab_binary_atab_binary30474ea5eaee4ec0e0a5a86377abb1ac on tab_binary
+        Index Cond: ((a)::bbf_binary = '0xbaadf00d'::bbf_binary)
+  Query Text: select * from tab_binary where a = 0xBAADF00D
+  ->  Index Scan using ix_tab_binary_atab_binary30474ea5eaee4ec0e0a5a86377abb1ac on tab_binary
+        Index Cond: ((a)::bbf_binary = '0xbaadf00d'::bbf_varbinary)
+  Query Text: select count(*) from tab_binary where a > 0xBAADF00D
+  ->  Aggregate
+        ->  Seq Scan on tab_binary
+              Filter: ((a)::bbf_varbinary > '0xbaadf00d'::bbf_varbinary)
+  Query Text: select count(*) from tab_binary where a < 0xBAADF00D
+  ->  Aggregate
+        ->  Seq Scan on tab_binary
+              Filter: ((a)::bbf_varbinary < '0xbaadf00d'::bbf_varbinary)
+  Query Text: select count(*) from tab_binary where a >= 0xBAADF00D
+  ->  Aggregate
+        ->  Seq Scan on tab_binary
+              Filter: ((a)::bbf_varbinary >= '0xbaadf00d'::bbf_varbinary)
+  Query Text: select count(*) from tab_binary where a <= 0xBAADF00D
+  ->  Aggregate
+        ->  Seq Scan on tab_binary
+              Filter: ((a)::bbf_varbinary <= '0xbaadf00d'::bbf_varbinary)
+~~END~~
+
+
+exec babel_3939_vu_prepare_p2;
+GO
+~~START~~
+text
+Query Text: EXEC babel_3939_vu_prepare_p2
+  Query Text: select * from tab_varbinary where a = 0xBAADF00D
+  ->  Index Scan using ix_tab_binary_btab_varbinary785ec2330c9a18520469e01cd1dc6f53 on tab_varbinary
+        Index Cond: ((a)::bbf_varbinary = '0xbaadf00d'::bbf_varbinary)
+  Query Text: select * from tab_varbinary where a = cast(0xBAADF00D as  binary )
+  ->  Index Scan using ix_tab_binary_btab_varbinary785ec2330c9a18520469e01cd1dc6f53 on tab_varbinary
+        Index Cond: ((a)::bbf_varbinary = '0xbaadf00d'::bbf_binary)
+  Query Text: select * from tab_varbinary where a > cast(0xBAADF00D as  binary )
+  ->  Seq Scan on tab_varbinary
+        Filter: ((a)::bbf_varbinary > '0xbaadf00d'::bbf_varbinary)
+  Query Text: select * from tab_varbinary where a < cast(0xBAADF00D as  binary )
+  ->  Seq Scan on tab_varbinary
+        Filter: ((a)::bbf_varbinary < '0xbaadf00d'::bbf_varbinary)
+  Query Text: select * from tab_varbinary where a >= cast(0xBAADF00D as  binary )
+  ->  Seq Scan on tab_varbinary
+        Filter: ((a)::bbf_varbinary >= '0xbaadf00d'::bbf_varbinary)
+  Query Text: select * from tab_varbinary where a <= cast(0xBAADF00D as  binary )
+  ->  Seq Scan on tab_varbinary
+        Filter: ((a)::bbf_varbinary <= '0xbaadf00d'::bbf_varbinary)
+~~END~~
+
+
+select set_config('enable_bitmapscan', 'on', false);
```

Comment:
Please put `SET BABELFISH_SHOWPLAN_ALL OFF;` before this. Otherwise, this statement won't be executed.

Reply:
resolved